src/utils.py

```diff
@@ -1118,9 +1120,8 @@ def generate_token_OTP(mobile, request_header, kvdb_bucket):
             print("OTP SMS is:" + response.text)
             print("OTP SMS len is:" + str(len(response.text)))
 
-            OTP = response.text
-            OTP = OTP.replace("Your OTP to register/access CoWIN is ", "")
-            OTP = OTP.replace(". It will be valid for 3 minutes. - CoWIN", "")
+            sms_regex = r"(?<!\d)\d{6}(?!\d)"
```

Comment:
Can you please move this variable at the top near other constants?
```suggestion
            SMS_REGEX = r"(?<!\d)\d{6}(?!\d)"
```